client/src/components/quantity.tsx

```diff
@@ -0,0 +1,26 @@
+import { useState } from "react";
+
+export default function Quantity() {
+  const [count, setCount] = useState(0);
+  return (
+    <>
+      <div>
+        <label> Quantity </label>
+
+        <div>
+          {count > 0 && (
+            <button type="button" onClick={() => setCount(count - 1)}>
```

Comment:
one thing on the set function, if you want to derive a value from existing state (count in this case), you should pass in a function to the set function... example ```setCount(c => c + 1)``` where c is the current count. reference: https://react.dev/reference/react/useState#setstate